public_html/wp-content/plugins/wordcamp-speaker-feedback/wordcamp-speaker-feedback.php

```diff
@@ -55,16 +62,52 @@ function load() {
 	require_once get_includes_path() . 'spam.php';
 	require_once get_includes_path() . 'stats.php';
 	require_once get_includes_path() . 'view.php';
-
 	require_once get_includes_path() . 'admin.php';
+
+	add_action( 'init', __NAMESPACE__ . '\add_support', 99 );
+	add_action( 'init', __NAMESPACE__ . '\add_page_endpoint' );
+	add_action( 'rest_api_init', __NAMESPACE__ . '\register_rest_routes', 100 );
+
+	// Check if the page exists, and add it if not.
+	add_action( 'init', __NAMESPACE__ . '\add_feedback_page' );
 }
 
 /**
  * Create main Feedback page.
  *
+ * @param bool $is_network True if activating network-wide.
+ *
  * @return void
  */
-function activate() {
+function activate( $is_network = false ) {
+	if ( $is_network ) {
+		activate_on_network();
+	} else {
+		activate_on_current_site();
+	}
+}
+
+/**
+ * Run the activation routine on all valid sites in the network.
+ *
+ * @return void
+ */
+function activate_on_network() {
+	$valid_sites = get_site_ids_without_skip_flag();
+
+	foreach ( $valid_sites as $blog_id ) {
+		switch_to_blog( $blog_id );
+		activate_on_current_site();
+		restore_current_blog();
+	}
+}
+
+/**
+ * The activation routine for a single site.
+ *
+ * @return void
+ */
+function activate_on_current_site() {
```

Comment:
I think this should also check for `can_load` or at least the `has_dependency` check — the Leave Feedback page was created on central in my testing.

Reply:
I think there was a reason I didn't do the check here, but I can't remember it now...

🤔 Did you add the skip flag to Central before network activating? Theoretically that should prevent the page from getting created there...

Reply:
I didn't think to do that, since I assumed the lack of WC Post Types would prevent it — but if you'll set it on central on production, then it's fine.